web/ui/dashboard/src/app/guards/iframe/iframe.guard.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { CanActivate } from '@angular/router';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class IframeGuard implements CanActivate {
+	canActivate(): boolean {
+		const isIframe = window.self !== window.top;
+		if (isIframe) return false;
```

Comment:
Seems like this can be simplified to 
```suggestion
		return window.self === window.top;
```